examples/storybook/src/wrap-with-hits.ts

```diff
@@ -101,7 +101,7 @@ export function wrapWithHits({
 
     constructor() {
       Object.keys(methods).forEach(methodName => {
-        this[methodName] = methods[methodName];
+        (this as any)[methodName] = methods[methodName];
```

Comment:
Can you add the methods in the top as `[method in keyof TMethods]: TMethods[method]`, or does that not work?